README.md

```diff
@@ -361,6 +362,41 @@ const ColumnAdder = ({ addColumn }) {
 </Board>
 ```
 
+#### `renderCardAdder`
+
+Use this if you want to render your own card adder. You have to pass a function and return your card adder component.
+The function will receive these parameters:
+
+| Arg          | Description                                          |
+| -----------  | ---------------------------------------------------  |
+| `column`     | the column wich the card is being added              |
+
+| function     | Description                                                |
+| ------------ | ---------------------------------------------------------- |
+| `onConfirm`  | Call this function with a new card to add the new card |
+
+\* It's unavailable when the board is controlled.
```

Comment:
Why are your thoughts to not making this available to the controlled one?

Reply:
I think it could be done in another PR, i dont know how time it would be needed to make it on controlled too.

If you think we could do it in this PR, give me some directions